### PR TITLE
Sprint2 db entity

### PR DESCRIPTION
**Pull request by vladvorontsov**
Updating of schema and insert script, deleting of unnecessary entities, modification of entity "group" (adding onetomany relationship with annotation @ElementCollection to implementation of set of experts). Pay attention please that there is some mistake in class UserRepositoryImpl in package repository/custom, there is some undefined class QUser.

### CODE REVIEW

File: src/main/resources/schema.sql [hunk 1]
```diff
@@ -2,209 +2,181 @@ SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
...
 ENGINE = InnoDB;
 
-DROP TABLE IF EXISTS `educational_group`;
-CREATE TABLE `educational_group` (
+-- -----------------------------------------------------
+-- Table `education_group`
+-- -----------------------------------------------------
+
+DROP TABLE IF EXISTS `education_group`;
+CREATE TABLE `education_group` (
```
Comment (cdc895642) [inline src/main/resources/schema.sql]: need at least another field "budgetOwner". Look at Story 4

Comment (kmusienko) [inline src/main/resources/schema.sql]: See my changes. I have already fixed it.

Comment (kmusienko) [inline src/main/resources/schema.sql]: https://github.com/Crutovel/SoftServe-TeamProject/commit/fd423095c4deadb62f75d6d0f9100f410e2bad22

**Comment by khdimon** [comment]
Maybe "Specialization" could be changed to "Direction" according to stories.

**Pull request merged**